benchmark/utils.go

```diff
@@ -55,3 +56,55 @@ func GetImageListFromCsv(csvLoc string) ([]ImageDescriptor, error) {
 	}
 	return images, nil
 }
+
+func GetCommitHash() (string, error) {
+	cmd := exec.Command("git", "rev-parse", "HEAD")
+	output, err := cmd.Output()
+	if err != nil {
+		return "", err
+	}
+	return string(output), nil
+}
+
+func GetDefaultWorkloads() []ImageDescriptor {
+	var imageList []ImageDescriptor
+	imageList = append(imageList,
```

Comment:
Why append instead of just returning a slice literal?

Reply:
Agreed. Changed to use slice literal